src/Nethermind/Nethermind.BeaconNode/MockedStart/QuickStart.cs

```diff
@@ -92,12 +80,8 @@ public void QuickStartGenesis()
             _logger.LogWarning(0, "Mocked quick start with genesis time {GenesisTime} and {ValidatorCount} validators.",
```

Comment:
there is one imporrant thing that we have been doing - before every Log call we check if logging is enables for rhis level -have a look at Nethermind calls

Reply:
So, checking logging levels is not really relevant for normal log calls; the only exception are ones like above where I do ToString() which, yes, you can avoid evaluating through a guard.

In this particular case, it is only start up code, and only for mocked test scenarios, so I didn't worry about the performance. But consistency is important, so adding a guard makes sense (for if a future developer copies the code elsewhere).

Another approach is to support format specifiers, for the shortstring example, it could be:

`_logger.LogDebug("Quick start adding deposit for mocked validator {ValidatorIndex} with public key {PublicKey:T10}.", validatorIndex, publicKey);
`
Then interpret the format T10 as Truncate to 10. This avoids any performance hit as ToString is only called if the message is actually logged.